DevSpaceWeb/Properties/PublishProfiles/Github.pubxml

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+    <ContainerRepository>Misha-133/DevSpace/Web</ContainerRepository>
```

Comment:
does this change where the package is published to?

Reply:
correct. You can't just push to some arbitrary repository too, as you are limited by `GITHUB_TOKEN`s scope